CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# How to contribute to JuMP
+
+Welcome! This document explains some of the ways you can contribute to JuMP.
+
+## Join the Discourse forum
+
+First up, join the [Discourse forum](https://discourse.julialang.org/c/domain/opt).
+
+The forum is a good place to ask questions about how to use JuMP. You can also
+use the forum to discuss possible feature requests and bugs before raising a
+Github issue (more on this below).
+
+Aside from asking questions, the easiest way you can contribute to JuMP is to
+help answer questions on Discourse!
+
+## File a bug report
+
+Another way to contribute to JuMP is to file [bug reports](https://github.com/JuliaOpt/JuMP.jl/issues/new?template=bug_report.md).
+
+Before you do however, make sure you read the following before posting a new bug
+report:
+
+- If you have a question or are unsure if the behavior you're experiencing is a
+  bug, please search or post to our [Discourse site](https://discourse.julialang.org/c/domain/opt).
+  Questions posted to Discourse have broader visibility and are likely to be
+  answered more quickly than Github issues.
+
+- If you're experiencing a bug that is solver-specific (e.g., it only happens
```

Comment:
Not a native but I have read that "you are" should be used rather than "you're" in written English